hw-25.js

```diff
@@ -0,0 +1,111 @@
+
+
+const delay = (ms) => new Promise(resolve => setTimeout(resolve, ms));
+
+const randomResolve = async () => {
+  try {
+    const randomDecimal = Math.random();
+    const rand = await delay(1000);
+
+    if (randomDecimal < 0.5) {
+      return Promise.resolve("even");
+    } else {
+      return Promise.reject("odd");
+    }
+  } catch (error) {
+    throw error;
+  }
+};
+
+randomResolve()
+  .then(result => {
+    console.log(result);
+  })
+  .catch(error => {
+    console.error(error);
+  });
+
+
+  
+
+
+const delay = (ms) => new Promise (resolve => setTimeout(resolve, ms));
```

Comment:
looks like error
why did you need to create new function with same name and same logic?